app/controllers/events_controller.rb

```diff
@@ -41,6 +41,10 @@ def create
   def edit
   end
 
+  def show
+    @booking = Booking.where(event: @event, applicant: current_user).first if current_user
```

Comment:
Noting that `Booking.where(event: @event, applicant: current_user).first` can also be written as `Booking.find_by(event: @event, applicant: current_user)`